Anlab.Core/Models/OrderDetails.cs

```diff
@@ -23,6 +23,11 @@ public class OrderDetails
 
         public string LabComments { get; set; }
         public decimal AdjustmentAmount { get; set; }
+        public decimal GrandTotal { get

```

Comment:
@srkirkland I was just using the Total before. Didn't see a need to create a new field. I'm fine either way.